zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -132,32 +142,36 @@ object Request {
     val h  = headers
     val ra = remoteAddress
     val d  = data
+    val v  = version
 
     new Request {
       override def method: Method                     = m
       override def url: URL                           = u
       override def headers: Headers                   = h
+      override def version: Version                   = v
       override def unsafeEncode: HttpRequest          = {
-        val jVersion = Version.`HTTP/1.1`.toJava
+        val jVersion = v.toJava
         val path     = url.relative.encode
         new DefaultFullHttpRequest(jVersion, method.toJava, path)
       }
       override def remoteAddress: Option[InetAddress] = ra
       override def data: HttpData                     = d
+
     }
   }
 
   /**
    * Effectfully create a new Request object
    */
   def make[E <: Throwable](
+    version: Version = Version.`HTTP/1.1`,
     method: Method = Method.GET,
     url: URL = URL.root,
     headers: Headers = Headers.empty,
     remoteAddress: Option[InetAddress],
     content: HttpData = HttpData.empty,
   ): UIO[Request] =
```

Comment:
Do we need this method?

Reply:
This is not used Yet but when we merge client and server request this might be useful for clients

Reply:
Deleted as was not getting used